src/main/java/com/tourism/psk/model/GroupTrip.java

```diff
@@ -1,13 +1,15 @@
 package com.tourism.psk.model;
 
 import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
 import java.util.Objects;
 import java.util.Set;
 
 @Entity
-public class GroupTrip {
+public class    GroupTrip {
```

Comment:
According to swagger, should also have a status field (enum: [ pending, approved, archived ]);